src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -171,15 +171,11 @@ public void jobLogic() {
                 status.fail("Could not clone version.", e);
             }
         }
-
         // FIXME: Should we overwrite the input GTFS dataset if transforming in place?
-
-        // TODO: Any other activities that need to be run (e.g., module-specific activities).
-        if (DataManager.isModuleEnabled("deployment")) {
-//            Project project = feedSource.retrieveProject();
-//            if (project.autoDeployId)
-            // TODO: Get deployment, update feed version for feed source, and kick off deployment to server that
-            //  deployment is currently pointed at.
+        // If deployment module is enabled and feed source is deployable, add auto deploy job. Note: other checks occur
+        // within job to ensure appropriate conditions are met.
+        if (DataManager.isModuleEnabled("deployment") && feedSource.deployable){
```

Comment:
I can't recall if we discussed this fully, but it might be better to add an enum set instead of a boolean for autoDeploy. That way, we can be more precise about when auto deploy should kick off. Example values could be, `ON_FEED_FETCH`, `DAILY`, `ON_PROCESS_FEED`, and maybe some others. It would make sense to handle those checks at this level rather than inside of the AutoDeployJob itself.

Reply:
Updated.